securebanking-openbanking-uk-rcs-server/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/testsupport/TestInitializer.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.securebanking.openbanking.uk.rcs.testsupport;
+
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import static com.forgerock.securebanking.openbanking.uk.rcs.testsupport.WireMockServerExtension.SERVER_PORT;
+
+/**
+ * Replaces URLs within the test configuration with the wiremock instance.
+ */
+public class TestInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+    @Override
+    public void initialize(ConfigurableApplicationContext applicationContext) {
+        TestPropertyValues values = TestPropertyValues.of(
+                "am.endpoint.userprofile=http://localhost:" + SERVER_PORT + "/am/userprofile",
+                "rs.base-url=http://localhost:" + SERVER_PORT + "/rs",
+                "idm.base-url=http://localhost:" + SERVER_PORT + "/idm",
+                "jwkms.base-url=http://localhost:" + SERVER_PORT + "/jwkms",
+                "jwkms.base-url=http://localhost:" + SERVER_PORT + "/jwkms",
+                "am.root=http://localhost:" + SERVER_PORT + "/am/oauth2/authorize"
```

Comment:
This overrides the static Spring test configuration, meaning that requests for these services will be sent to WireMock.